README.md

```diff
@@ -51,8 +51,9 @@ circular, ok := obj.(*Circular)
 
 ## Customize Usage Examples
 
-#### Struct filed name encoding
-Hessian encoder will convert the filed name of struct to lower camelcase defaulted, but you customize it by `hessian` tag of struct.
+#### Encoding filed name 
+
+Hessian encoder default converts filed names of struct to lower camelcase, but you can customize it using `hessian` tag.
```

Comment:
but you can customize it by using `hessian` tag. or but you can customize it by `hessian` tag.

Reply:
the syntax `customize something using something` is also correct.